pkg/logentry/stages/template.go

```diff
@@ -34,8 +36,12 @@ var (
 		"TrimPrefix": strings.TrimPrefix,
 		"TrimSuffix": strings.TrimSuffix,
 		"TrimSpace":  strings.TrimSpace,
-		"Sha256": func(salt string, s string) string {
-			hash := sha256.Sum256([]byte(salt + s))
+		"Sha2Hash": func(salt string, input string) string {
+			hash := sha256.Sum256([]byte(salt + input))
+			return hex.EncodeToString(hash[:])
+		},
```

Comment:
```suggestion
                "Hash": func(salt string, input string) string {
			hash := sha3.Sum256([]byte(salt + input))
			return hex.EncodeToString(hash[:])
		},
		"Sha2Hash": func(salt string, input string) string {
			hash := sha256.Sum256([]byte(salt + input))
			return hex.EncodeToString(hash[:])
		},
```

Reply:
i think my spacing is screwed up you might not want to commit this, but just as an idea I think we could offer a default `Hash` which uses the sha3-256 under the hood

Reply:
yup. Fix in new commit